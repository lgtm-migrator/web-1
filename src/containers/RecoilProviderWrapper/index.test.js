import React from "react"
import renderer from "react-test-renderer"
import { RecoilProviderWrapper } from "./index"

describe("Recoil provider wrapper", () => {
  it("should render correctly with theme", () => {
    const tree = renderer
      .create(<RecoilProviderWrapper element={<>Test element</>} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
