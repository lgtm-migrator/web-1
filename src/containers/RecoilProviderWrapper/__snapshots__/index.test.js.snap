// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Recoil provider wrapper should render correctly with theme 1`] = `"Test element"`;
